File: CS/SpreadsheetExample.Module/BusinessObjects/SpreadsheetDocument.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace SpreadsheetExample.Module.BusinessObjects {
    public class SpreadsheetDocument :BaseObject {
        public SpreadsheetDocument(Session session) : base(session) { }

        public string Subject {
            get { return GetPropertyValue<string>("Subject"); }
            set { SetPropertyValue("Subject", value); }
        }

        [Delayed]
        [VisibleInListView(false)]
        public byte[] Content {
            get { return GetDelayedPropertyValue<byte[]>("Content"); }
            set { SetDelayedPropertyValue("Content", value); }
        }
    }
}

File: VB/SpreadsheetExample.Module/BusinessObjects/SpreadsheetDocument.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace SpreadsheetExample.Module.BusinessObjects
	Public Class SpreadsheetDocument
		Inherits BaseObject

		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Public Property Subject() As String
			Get
				Return GetPropertyValue(Of String)("Subject")
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Subject", value)
			End Set
		End Property

		<Delayed, VisibleInListView(False)>
		Public Property Content() As Byte()
			Get
				Return GetDelayedPropertyValue(Of Byte())("Content")
			End Get
			Set(ByVal value As Byte())
				SetDelayedPropertyValue("Content", value)
			End Set
		End Property
	End Class
End Namespace
